README.md

```diff
@@ -1,3 +1,15 @@
 # Community
 
 The Container Storage Interface (CSI) Community Reporsitory contains community related content that applies to all repositories in the CSI org.
+
+
+## Meeting
+
+We hold a public meeting every two weeks, on Wednesdays at 8 AM (PT).
+
+The meeting is held on Zoom: https://zoom.us/j/790748945
```

Comment:
Add a period in the end. Probably do this:
The meeting is held on [Zoom](https://zoom.us/j/790748945).